### DISCUSSION TITLE
gesture swipe for shortcut to the next/previous frame

### DISCUSSION THREAD

**Discussion by mamatism** [keiwando/toonsquid#51] [Ideas]
I really love toonsquid
good job!

btw, do you have a plan to add some gesture swipe finger to be the shortcut for the next/previous frame?
that would be nice and improve to faster my workflow.

**Comment by yaknepper** [comment]
I think this option is already available with a three-finger swipe. Tak a look in the settings menu.

Cheers

**Reply by mamatism** [reply]
thank you, you right, I got it now.

But, can we have an option to change three-finger swipe to only one finger swipe? because three-finger is difficult/not intuitive to use. 

it would be better if we can use one finger to swipe.


cheers

**Reply by yaknepper** [reply]
That would definitely make it smoother and more convenient, the three finger swipe can be difficult to hit right every time.

cheers
